wiki/Gameplay/Daily_challenge/en.md

```diff
@@ -0,0 +1,62 @@
+# Daily challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a daily streak by passing beatmaps consecutively with increasing difficulty with the difficulty being reset every 7 days.
+
+Every beatmap is handpicked from the [Featured Artist](/wiki/People/Featured_Artists) listing, and sometimes the maps come with a mod forced onto users, which requires them to pass the map with the selected mod.
+
+## Game menu
+
+From the main menu, the daily challenge menu can be accessed with the following steps:
+
+1. Click the `play` button or press `P`.
+2. Click the `daily challenge` button or press `D`.
+
+![](img/daily-challenge-menu.png "Screenshot of the daily challenge menu during November 7, 2024.") 
+
+On entering, the user is presented with an intro showing the beatmap the user has to pass, and the mods they have to pass it with.
+
+After the intro ends, the user can view the data for today's challenge. The left side displays various score-related information like the total pass count and the cumulative total score. In the middle, a leaderboard shows the top scores achieved by players. Users are able to discuss the daily challenge in the chat on the right.
+
+## Beatmap difficulty range
+
+![](img/Beatmap-selection-criteria.png "Specific star ratings beatmaps have to follow for the entire week")
+
+## Streak milestones
+
+Play streaks are displayed on the user profile with different colours depending on certain streak milestones.
+
+### Tier breakdown
+
+<!-- tier images: https://www.figma.com/design/tc79qAgJ35KQvdTO0Oj3dN/Daily-Challenge-Counter?node-id=0-1&t=xjRm9Ke0tUMtAQlh-1 -->
+
+| | Tier | Total participation | Daily streak | Weekly streak |
```

Comment:
```suggestion
|  | Tier | Total participation | Daily streak | Weekly streak |
```

This is an oversight and not due to the images, right?

Reply:
it's an intentional choice to make the images properly aligned in their own column, we've done this kind of thing before iirc

Reply:
oh yea it's causing a CI fail because of the missing space, will fix